pom.xml

```diff
@@ -65,6 +65,7 @@
         <graphvizapi.version>1.1</graphvizapi.version>
         <groovy.version>2.4.7</groovy.version>
         <guava.version>20.0</guava.version>
+        <jackson.version>2.7.3</jackson.version>
```

Comment:
The version used by ipst should be adapt to be compatible with ipst-core. This adaptation should be done in ipst first.